test/integration/framework/assets/00-custom-resource-definition.yaml

```diff
@@ -0,0 +1,1137 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
We don't necessarily need to check these in, but if we want a hermetic integration test harness we should. TODO on my end to get a CI job running the integration test.